modin/pandas/dataframe.py

```diff
@@ -1966,19 +1965,8 @@ def __setattr__(self, key, value):
 
     def __setitem__(self, key, value):
         if hashable(key) and key not in self.columns:
-            # Handle new column case first
-            if isinstance(value, Series):
-                if len(self.columns) == 0:
-                    self._query_compiler = value._query_compiler.copy()
-                else:
-                    self._create_or_update_from_compiler(
-                        self._query_compiler.concat(
-                            1,
-                            value._query_compiler,
-                            join="left",
-                        ),
-                        inplace=True,
-                    )
```

Comment:
Why removed?

Reply:
now in this case is processed by `DataFrame.insert`